src/components/elements/RouterLink.tsx

```diff
@@ -24,7 +24,12 @@ const RouterLink = forwardRef<RouterLinkProps, any>(
       target={openInNew ? '_blank' : undefined}
     >
       {openInNew || Icon ? (
-        <Stack direction={'row'} gap={0.75} alignItems='center'>
+        <Stack
+          direction={'row'}
+          gap={0.75}
+          alignItems='center'
+          sx={{ display: 'inline-flex', textDecoration: 'inherit' }}
```

Comment:
Note of interest, this `textDecoration` is now necessary because text decorations do not propagate to atomic inline-level descendants 😂  https://bugzilla.mozilla.org/show_bug.cgi?id=1344545